contracts/fixed_price_rev1.1.scilla

```diff
@@ -131,7 +129,12 @@ let make_error =
 
 contract FixedPriceOrderbook
 (
-  initial_contract_owner: ByStr20
+  initial_contract_owner: ByStr20,
+  collection_contract: ByStr20 with contract
```

Comment:
Does it make sense to have a transition that would allow the contract owner/admin to update the collection contract?

Reply:
Yes, added with https://github.com/Zilliqa/marketplace-contracts/pull/54/commits/a5d6de5e7defcbe9cfd98e16d0ba2d8c989ca573